.gitignore

```diff
@@ -24,6 +24,8 @@ website/node_modules
 *.iml
 *.test
 *.iml
+powermax-go-client
+terraform-provider-powermax
 
```

Comment:
Why ignoring terraform-provider-powermax?

Reply:
@forrestxia why do we need this to be pushed? we will be building it with the code.